src/dparse/ast.d

```diff
@@ -2429,6 +2430,18 @@ public:
     mixin OpEquals;
 }
 
+final class ParameterAttribute : ASTNode
+{
+    override void accept(ASTVisitor visitor) const
+    {
+        mixin (visitIfNotNull!(atAttribute));
+    }
+
+public:
```

Comment:
Isn't this redundant? I think that the default for class members is `public`. Putting it here just implies that `accept` isn't `public`.

Reply:
Removed. I will remove them from all the ASTNodes later